core/models.py

```diff
@@ -159,7 +170,8 @@ def is_saved(self: "Post", user: User) -> bool:
     def get_comments(self: "Post") -> models.QuerySet:
         return self.children.all()
 
-    def get_comment_form(self: "Post") -> CommentForm:
+    def get_comment_form(self: "Post"):

```

Comment:
Why this change? I mean, removing type?

Reply:
I had a problem with circular imports (models from forms, forms from models), so I moved this import only to import in function. When I moved it, the type throws an error.

Reply:
I am aware of it. 
What I pointed out was removing type, not the import.

Reply:
Because when import was inside the function, the typing gave an error.

Reply:
oh, interesting. to keep types in place you can add `-> any:` there. Should solve issue and code style is consistent.

Reply:
No, ruff doesn't allow type Any here!

Reply:
![image](https://github.com/pymasterspl/reddit/assets/989256/c599cc8a-05a8-4377-909f-2bad3ab1d64f)
Sorry, small caps. 
This works in other PR.
Also please remove this TYPE_CHECKING check.

Reply:
Corrected. Thanks :)